<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1326</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_SaveSlots">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>&lt; Save slot 1 &gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt; Save slot 2 &gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt; Save slot 3 &gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt; Save slot 4 &gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt; Save slot 5 &gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt; Save slot 6 &gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt; Save slot 7 &gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt; Save slot 8 &gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt; Save slot 9 &gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt; Save slot 10 &gt;</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Marroses">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>201</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_AddMacros">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_DeleteMacros">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_MacrosType">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Equipment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Magic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Items</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Built-in</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>done-by-myself hotkey</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hotkeys group</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox_MacrosKeyShift">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>+ Shift</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_MacrosKeyAlt">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>160</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>+ Alt</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_MacrosKeyCtrl">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>+ Ctrl</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Save slots:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hotkey type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key:</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_MacrosKey">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_Pages">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>1021</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="Equipment">
     <attribute name="title">
      <string>Equipment</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Magic">
     <attribute name="title">
      <string>Magic</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Items">
     <attribute name="title">
      <string>Items</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="DIY">
     <attribute name="title">
      <string>DIY</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="BuiltIn">
     <attribute name="title">
      <string>Built-In</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="HotkeysGroup">
     <attribute name="title">
      <string>Hotkeys group</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Settings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QSlider" name="horizontalSlider_StandardPause">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>450</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>420</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Standard pause time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Manual or auto</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>70</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Controls of Elden Ring</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>150</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Attack</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>200</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Strong attack</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>250</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Block</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>300</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Weapon art</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>350</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Next magic</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>400</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Next item</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>440</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>490</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Crouch</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_OpenSaveFile">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_SaveSettings">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
